daprdocs/content/en/operations/components/component-secrets.md

```diff
@@ -88,12 +57,14 @@ spec:
     	name: redis-secret
         key:  redis-password
 auth:
-  secretStore: kubernetes
+  secretStore: <SECRET_STORE_NAME>
 ```
 
-The above example tells Dapr to use the `kubernetes` secret store, extract a secret named `redis-secret` and assign the value of the `redis-password` key in the secret to the `redisPassword` field in the Component.
+When running in Kubernetes and using a Kubernetes secret store, either the field `auth.SecretStore` can be empty (as it is assumed to be Kubernetes secret store) or it needs to be `kubernetes`. For all other secret store, the `SECRET_STORE_NAME` is the name of the configured secret store component.
+
+The above example tells Dapr to extract a secret named `redis-secret` from the defined secret store and assign the value of the `redis-password` key in the secret to the `redisPassword` field in the Component.
 
-### Creating a secret and referencing it in a Component
+### Creating a Kubernetes secret and referencing it in a Component
```

Comment:
Component should be lowercase

Reply:
This is not a component. It is actually referring to the k8s platform.